pkg/storage/pod.go

```diff
@@ -101,7 +101,7 @@ func (s *podStorage) Store(newPods []PodMetricsPoint) {
 				continue
 			}
 			lastContainers[newContainer.Name] = newContainer
-			if newContainer.StartTime.Before(newContainer.Timestamp) && newContainer.Timestamp.Sub(newContainer.StartTime) < s.metricResolution {
+			if newContainer.StartTime.Before(newContainer.Timestamp) && newContainer.Timestamp.Sub(newContainer.StartTime) < s.metricResolution && newContainer.Timestamp.Sub(newContainer.StartTime) >= 10*time.Second {
```

Comment:
Please move 10*Seconds to const variable and add a comment why this value is used.

Reply:
ok, Let me do it now